Assets/Scripts/GhostController.cs

```diff
@@ -32,11 +36,20 @@ public class GhostController : MonoBehaviour {
 			transform.localScale = flipXScale;
 		}
 		if(Input.GetKey(KeyCode.Space)) {
+            //FIXME: do not keep the force increasing
 			body.AddForce (transform.up  * 10f);
 		}
 		if(Input.GetKey(KeyCode.W)) {
 			animator.SetBool ("isWind", true);
 			particles.Play ();
 		}
-	}
+
+        //let the main camera follow ghost.
+        Vector3 mainCameraOffset = new Vector3(0, 2.5f, -10f);
+        mainCameraPos.position = body.transform.position + mainCameraOffset;
```

Comment:
Not a change request, but a discussion

This following style leads a weird behaviour, the camera will change will quickly.
By using this way(setup a no-camera-movement) it will be a little better.
![image](https://user-images.githubusercontent.com/10692276/32634571-5f6832ba-c600-11e7-9e25-6b002c060a55.png)
But if we use this way, we also needs to add camera animation, otherwise, the position change will be a shock.